### DISCUSSION TITLE
Template and x-show/x-if don't render the content

### DISCUSSION THREAD

**Discussion by kesirog** [alpinejs/alpine#2995] [7. Bugs (archived)]
AlpineJS doesn't work when you try to render the content with x-show/x-if
```
<template x-show="condition">
 <div>something</div>
</template>
```

While x-show works completely fine on div element like this:
```
<div x-show="condition">
 <div>something</div>
</div>
```

**Comment by SimoTod** [comment]
X-show applies to the current element so you are toggling `display: none` on and off on a template tag. Template tags never show on page.

https://alpinejs.dev/directives/show
https://alpinejs.dev/directives/if
